src/main/java/xades4j/xml/unmarshalling/FromXmlCommitmentTypeConverter.java

```diff
@@ -79,7 +80,7 @@ public void convertFromObjectTree(
                             throw new PropertyUnmarshalException("Qualifiers with multiple children are not support", CommitmentTypePropertyBase.PROP_NAME);
                         }
 
-                        qualifiers.add(xmlQualifier.getContent().get(0));
+                        qualifiers.add((String) xmlQualifier.getContent().get(0));

```

Comment:
Should be `<Object>`

Reply:
we have to cast anyway later on, as it is used as a string. see other comment.